tools/ci_build/github/azure-pipelines/win-gpu-reduce-op-ci-pipeline.yml

```diff
@@ -16,32 +16,20 @@ jobs:
     EnvSetupScript: setup_env_cuda_11.bat
```

Comment:
Manually run this pipeline to validate your change.

Reply:
thank your reminder, reduce op ci needs flatbuffer